src/main/java/Zap.java

```diff
@@ -0,0 +1,252 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Zap {
+    private static List<Task> tasks = new ArrayList<>();
+    public static void main(String[] args) {
+        String logo = """
+              _____   _   _   ____
+             |__  /  / \\ / | |  _ \\
+               / /  / _ \\| | | |_) |
+              / /_ / ___ \\ | |  __/
+             /____/_/   \\_\\_| |_|
+            """;
+
+        System.out.println("Hello from\n" + logo);
+        greeting();
+        processCommands();
+        exit();
+    }
+
+    private static void greeting() {
+        String chatbotName = "ZAP";
+        System.out.println("____________________________________________________________");
+        System.out.println(" 你好! Hello! Konichiwa! I'm " + chatbotName);
+        System.out.println(" What can I do for you?");
+        System.out.println("____________________________________________________________");
+    }
+
+    /**
+     * main interaction loop of ZAP
+     * continuously prompts the user to enter commands and responds accordingly.
+     * The loop breaks when the user enters "thank you and bye."
```

Comment:
do not have to explain how the code works, just explain the what and why aspects of the code